Backend/Interview.Domain/Rooms/Service/RoomService.cs

```diff
@@ -204,6 +205,12 @@ public async Task<RoomPageDetail> CreateAsync(RoomCreateRequest request, Cancell
         var createdParticipants = await _roomParticipantService.CreateAsync(room.Id, participants, cancellationToken);
         room.Participants.AddRange(createdParticipants);
 
+        if (request.ScheduleStartTime is not null)
+        {
+            var scheduleStartTime = DateTimeOffset.FromUnixTimeSeconds(request.ScheduleStartTime.Value);
```

Comment:
Is it possible to get DateTimeOffset from the frontend? I.e. time together with offset and just convert it to Utc?

Reply:
the duration and start of the room are transmitted in the same style Unix sec.

Reply:
frontend -> Unix -> DateTimeOffset -> DateTime
DateTime -> DateTime -> frontend

Reply:
Frontend sends Unix time. It's time that have elapsed since 00:00:00 [UTC] on 1 January 1970. This time consider local timezone (converts local time to UTC). No need in dealing with timezones if we use Unix time. If I get you right